inc/ZipProvider.php

```diff
@@ -127,6 +127,31 @@ public function generate_zip_file() : bool {
 			return false;
 		}
 
+		// Build a mapping based on where the translation entries occur and separate the po entries.
+		$mapping    = $this->build_mapping( $entries );
+		$po_entries = array_key_exists( 'po', $mapping ) ? $mapping['po'] : [];
+		unset( $mapping['po'] );
+
+		// Create JSON file(s).
+		$json_file_base = "{$dest}/{$wp_locale}";
```

Comment:
`$dest` and `$wp_locale` are not defined. `$dest` should probably be provided by the `ZipProvider`.